index.html

```diff
@@ -17,22 +17,22 @@
         <span class="logo">Joy</span>
       </button>
   
-        <ul class="ul-nav" title="navigation-list">
-         <li><a href="#portfolio">Portfolio</a></li>
-         <li><a href="#about">About</a></li>
-         <li><a href="#contactme">Contact</a></li>
-        </ul>
-        
-      <button class="nav-button2">
-        <span> 
-          <img src="./sources/Hamburg" alt="hamburg">
-        </span>
-      </button>
+      <ul class="ul-nav" title="navigation-list">
+        <li><a class="navigationList" href="#portfolio">Portfolio</a></li>
+        <li><a class="navigationList" href="#about">About</a></li>
+        <li><a class="navigationList" href="#contactme">Contact</a></li>
```

Comment:
- [ ] as the previous reviewer suggested, kindly remove the white space on the right when the mobile menu pops to match the original design. Ensure that it covers the whole viewport width and height of the screen.

![image](https://user-images.githubusercontent.com/98453979/210757915-487dd4c0-c1de-4124-8247-b10fa0e51e05.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210758085-40933fdb-7da2-467a-a07d-7c967a0743d6.png)